src/accounts.rs

```diff
@@ -297,14 +299,20 @@ impl Accounts {
 /// Configuration file name.
 pub const CONFIG_NAME: &str = "accounts.toml";
 
+/// Lockfile name.
+pub const LOCKFILE_NAME: &str = ".lock";
+
 /// Database file name.
 pub const DB_NAME: &str = "dc.db";
 
 /// Account manager configuration file.
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug)]
 struct Config {
     file: PathBuf,
     inner: InnerConfig,
+    // We lock the lockfile in the Config constructors to protect also from having multiple Config
+    // objects for the same config file.
+    lock_task: Option<JoinHandle<anyhow::Result<()>>>,
```

Comment:
Another strange thing is that `drop()` takes a reference instead of a value. And if a type isn't `Default`, we need smth like this `Option`. Looks like better to always implement `Default` for low-level things like `JoinHandle`